fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+      parameters:
+      
+        - name: id
+          in: path
+          description: draft order `id`.
+          required: true
+          type: integer
+          format: int64
+        - name: draft_order
+          in: body
+          description: draft_order to create.
+          schema:
+            allOf:
+              - $ref: '#/definitions/DraftOrder'
+            required:
+            - line_items
+      responses:
+            
+      
+        '200':
+          description: Updated
+          schema:
+            $ref: '#/definitions/DraftOrder'
+        '202':
+          description: Accepted with Pending operations
+          schema:
+            $ref: '#/definitions/DraftOrder'
+          headers:
+            Location:
+              description: Polling URI endpoint.
+              type: string
+            Retry-After:
+              description: Polling frequency.
+              type: string
+        '400':
+          description: Bad Request
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+                  
+        '404':
+          description: Not Found
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'          
+    
+        '422':
+          description: Unprocessable Entity
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'     
+       
+        
+            
+    
+    delete:
+      description: Remove a draft order
+      tags:
+        - Delete
+      parameters:
+        - name: id
+          in: path
+          description: draft order id
+          required: true
+          type: integer
+          format: int64
+      responses:
+        '200':
+          description: Deleted draft order
+          schema:
+            $ref: '#/definitions/DraftOrder'
+            
+        '400':
+          description: Bad Request
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+                  
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X DELETE -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders/#{id}.json
+            
+        - lang: Ruby
+          source: |
+              require 'shopify_api'
+
+              shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+              ShopifyAPI::Base.site = shop_url
+              draft_order = ShopifyAPI::DraftOrder.find(#{id})
+              draft_order.destroy    
+      
+  /admin/draft_orders/count.json:
+    get:
+      summary: Return count of draft orders
+      description: This endpoint enables you to return a count of all draft orders.    
```

Comment:
Would it be accurate to go with "This endpoint returns a count..." instead of using the "enables you to..." construction?